tests/commands/put_bucket_acl.sh

```diff
@@ -14,6 +16,40 @@ put_bucket_acl_s3api() {
   return 0
 }
 
+reset_bucket_acl() {
+  if [[ $# -ne 1 ]]; then
+    log 2 "'reset_bucket_acl' requires bucket name"
+    return 1
+  fi
+  acl_file="acl_file"
+  if ! create_test_files "$acl_file"; then
```

Comment:
I'd like to see us move towards using the run/assert workflow provided in bats - which then also removes a bunch of raw calls to exit.

Reply:
Added some assert_failure calls to start in the latest test